tests/large/test_web.py

```diff
@@ -99,16 +99,18 @@ def test_interactive_language_select_install(self):
         self.expect_and_no_warn("Choose language:")
         self.child.sendline(install_language)
         self.expect_and_no_warn("Installation done", timeout=self.TIMEOUT_INSTALL_PROGRESS)
-        self.wait_and_no_warn()
+        self.wait_and_close()
         self.verify_install(install_language)
 
     def test_unavailable_language_select_install(self):
+        """Installing Firefox-dev in unavailable language should be rejected"""
         install_language = "ABCdwXYZ"
         self.child = pexpect.spawnu(self.command('{} web firefox-dev --lang={}'.format(UMAKE, install_language)))
         self.expect_and_no_warn("Choose installation path: {}".format(self.installed_path))
-        self.accept_default_and_wait(expect_warn=True)
-        self.child.close()
-        self.assertEqual(self.child.exitstatus, 1)
+        self.child.sendline("")
+        self.wait_and_close(expect_warn=True, exit_status=1)
+
+        self.assertFalse(self.launcher_exists_and_is_pinned(self.desktop_filename))
```

Comment:
Good catch! Thanks for adding the additional assert ;) (I wonder if in the future, we shouldn't factor this in an key_files_installed(), but that will be for later anyway)